wordgenerator.py

```diff
@@ -0,0 +1,110 @@
+import pygame
+import random
+import sys
+import math
+from copy import deepcopy
+
+CONST_FILENAME = "american-english.txt"  
```

Comment:
The all caps I think should imply that this is a CONST variable, I believe that's why nothing like `const` is included in python. `FILENAME` should suffice for this variable name.